app/views/profiles/_edit_private.haml

```diff
@@ -3,7 +3,13 @@
 -#   the COPYRIGHT file.
 
 %hr
-%h3= t('profiles.edit.your_private_profile')
+%h3{:class=>"inline"}
+  = t('profiles.edit.additional')
+= t('profiles.edit.additional_visibility_text')
+= check_box_tag 'profile[public_details]', true, profile.public_details, {'data-size' => 'mini', 'data-on-text' => t('profiles.edit.public'), 'data-off-text' => t('profiles.edit.private')}
+%span{ :rel => 'facebox', :title => t('profiles.edit.additional_hint') }
```

Comment:
In most cases: yes. In this case I am not sure. What should the facebox do? AFAIR faceboxes had some URL before to load the content which is missing here. Depending on the content you could use a modal, a [tooltip](http://getbootstrap.com/javascript/#tooltips) or a [popover](http://getbootstrap.com/javascript/#popovers).